tasks/config/copy.js

```diff
@@ -3,6 +3,7 @@ module.exports = function () {
     devSource: {
       options: { mode: true },
       src: [
+        'yarn.lock',
```

Comment:
Makes sense to add yarn.lock to the build, but we should also remove it after installing dependencies.

Reply:
++ done